requirements.txt

```diff
@@ -1,4 +1,7 @@
 # generated from manifests external_dependencies
+# The factur-x library requires pypdf >=3.15.0
+# pypdf 5.0 dropped support for python<=3.7
+pypdf>=3.15.0,<5.0
```

Comment:
you shouldn't edit requirements.txt directly. Use test-requirements for that

Reply:
Thanks @etobella ,
I updated it.